src/locales/fr/menu.ts

```diff
@@ -44,10 +44,114 @@ export const menu: SimpleTranslationEntries = {
   "score": "Score",
   "wave": "Wave",
   "loading": "Chargement…",
-  "loadingAsset": "Loading asset: {{assetName}}",
+  "loadingAsset": "Chargement des ressources: {{assetName}}",
   "playersOnline": "Joueurs Connectés",
   "yes":"Oui",
   "no":"Non",
   "disclaimer": "AVERTISSEMENT",
-  "disclaimerDescription": "Ce jeu n’est pas un produit fini et peut contenir des problèmes de jouabilité, dont de possibles pertes de sauvegardes,\ndes modifications sans avertissement et pourrait ou non encore être mis à jour ou terminé."
+  "disclaimerDescription": "Ce jeu n’est pas un produit fini et peut contenir des problèmes de jouabilité, dont de possibles pertes de sauvegardes,\ndes modifications sans avertissement et pourrait ou non encore être mis à jour ou terminé.",
+  "general": "Général",
+  "display": "Affichage",
+  "audio": "Audio",
+  "gamepad": "Manette",
+  "keyboard": "Clavier",
+  "gameSpeed": "Vitesse du jeu",
+  "hpBarSpeed": "Vitesse de la barre de PV",
+  "expGainsSpeed": "Vitesse des gains d’EXP",
+  "expPartyDisplay": "Afficher EXP équipe",
+  "skipSeenDialogues": "Passer les dialogues déjà vus",
+  "battleStyle": "Style de combat",
+  "enableRetries": "Activer les réessais",
+  "tutorials": "Tutoriels",
+  "touchControls": "Contrôles tactiles",
+  "vibrations": "Vibrations",
+  "normal": "Normal",
+  "fast": "Rapide",
+  "faster": "Plus rapide",
+  "skip": "Passer",
+  "levelUpNotifications": "Montée de niveau",
+  "on": "Activé",
+  "off": "Désactivé",
+  "switch": "Remplacer",
```

Comment:
```suggestion
  "switch": "Choix",
```
From the beginning, the French game modes have always been "Choix" and "Défini"